scripts/kafka_base_dir.sh

```diff
@@ -0,0 +1,6 @@
+if [ "$TRAVIS" == "true" ]
```

Comment:
also `[` doesn't work with `==` for equality see: http://mywiki.wooledge.org/BashFAQ/031